### ISSUE TITLE
In iOS build flutter logs are not printing

### ISSUE THREAD

**Issue by jigarpatel17** [appspector/flutter-plugin#15]
Hi, In iOS build the print statements of is not logging in AppSpector console, only native print statements are logging. When i am running app directly from XCode it's working fine but from build it's not(Tried with both debug and release build).

**Comment by deszip** [comment]
Hi @jigarpatel17, thanks for reporting this. We'll take a look, looks like flutter/SDK interconnection issue. Could you please tell what iOS/Flutter versions are you on?

**Comment by jigarpatel17** [comment]
Thanks @deszip for quick response
AppSpector:- 0.0.9 and XCode:- 11.1

**Comment by jigarpatel17** [comment]
@deszip Any update?

**Comment by deszip** [comment]
Yep, we found the root cause, fixed it, still requires a bit of testing. Hopefully will be in master by the end of the day.

**Comment by jigarpatel17** [comment]
@deszip Any update when it will be in master branch?

**Comment by deszip** [comment]
Actually it's already there, we need some more testing and will push plugin to Flutter repo after that.
I'll tag you here when it'll be ready. Thanks for your patience.

**Comment by deszip** [comment]
@jigarpatel17 plugin v 0.1.0 published. It contains fix for the logging issue. Would be great if you can verify it works in your case.

**Comment by jigarpatel17** [comment]
@deszip I updated version to 0.1.0 and also update pod file but still it's not working

**Comment by deszip** [comment]
I'll keep looking into this, thanks. Would be great if you could show example code you use for logging, and explain what do you mean by 'from build'?. That's when you run `flutter run`?

**Comment by jigarpatel17** [comment]
> what do you mean by 'from build'?. That's when you run flutter run?
Generated diawi link from ipa file(In debug and also tried in release).

**Comment by deszip** [comment]
Working on this, will keep you updated.

**Comment by jigarpatel17** [comment]
@deszip I have to publish my app. Can you fix it as soon as possible?

**Comment by deszip** [comment]
Trying to do our best, looks like this issue is somehow connected with diawi. Would be great if you can check if logging works when you install ipa via TestFlight.

Sorry for inconvenience, we'll raise priority of this issue and will ping you when we'll make progress.

**Comment by sergeyzenchenko** [comment]
Hi @jigarpatel17 Where do you want to publish it?

**Comment by jigarpatel17** [comment]
@sergeyzenchenko In app store

**Comment by jigarpatel17** [comment]
> Trying to do our best, looks like this issue is somehow connected with diawi. Would be great if you can check if logging works when you install ipa via TestFlight.

Sorry for inconvenience, we'll raise priority of this issue and will ping you when we'll make progress.

Okay @deszip let me check. Will let you know once i have done.

**Comment by deszip** [comment]
Are you sure you are not on a release build? SDK disables itself when detects release environment.

**Comment by nobimac7449** [comment]
I also got this issue with debug build

**Comment by mscherer82** [comment]
Any update on this?

**Comment by deszip** [comment]
Unfortunately still nothing, would be great if you can build a minimal test project to demonstrate logs issue. We were unable to reproduce it on our builds.

**Comment by tomas-carv-com** [comment]
Same here, logs does not work with firebase distribution build
